src/sales/credit-memo-create.md

```diff
@@ -1,115 +1,179 @@
 ---
-title: Issuing a Credit Memo
+title: Create a Credit Memo
 ---
 
-Before a credit memo can be printed, it must first be generated for the order. Any credit memo with a [status]({% link sales/order-status.md %}) of “open” has an outstanding refund due. {% if "Default.EE-B2B" contains site.edition %}If you create a credit memo for an order that includes gift options, the refund for the gift wrapping and/or printed card appears in the Refund Totals section of the credit memo. To exclude these costs from the amount to be refunded, enter the amount as an Adjustment Fee. If multiple credit memos are issued for the same order, the refund for gift options appears in only the first credit memo.{% endif %}
+Before a credit memo can be printed, it must first be generated for an invoiced order. You can issue both online and offline refunds from an open credit memo, depending on the payment method. <!--{% if "Default.EE-B2B" contains site.edition %}-->Refunds can also be applied to store <!--{% if "Default.B2B Only" contains site.edition %}--> or company <!--{% endif %}-->credit.<!--{% endif %}-->
+
+- Purchases made by credit card can be refunded either online or offline.
+- Purchases made by check or money order must be refunded offline.
+
+Any credit memo with a [status]({% link sales/order-status.md %}) of “open” has an outstanding refund due. The refund workflow for orders paid by credit card is determined by the Payment Action setting in the configuration of the payment method. Refunds cannot be issued until the transaction is settled.
+
+- If Payment Action is set to `Authorize`, you must first generate the invoice from the Magento Admin before a credit memo can be created.
+- If Payment Action is set to `Authorize and Capture`, the invoice has already been  generated by the payment processor, but the funds aren't available until the transaction is settled. This brief waiting period is recommended by many payment processors as a security measure, and can usually be handled automatically.Transactions can also be settled manually from your merchant account with the payment processor.
```

Comment:
```suggestion
- If Payment Action is set to `Authorize and Capture`, the invoice has already been generated by the payment processor, but the funds aren't available until the transaction is settled. This brief waiting period is recommended by many payment processors as a security measure, and can usually be handled automatically. Transactions can also be settled manually from your merchant account with the payment processor.
```

Reply:
A couple of spacing issues here:
- There is an extra space between "been  generated"
- There is no space between "automatically.Transactions"